app/src/main/java/app/accrescent/client/receivers/AppInstallBroadcastReceiver.kt

```diff
@@ -49,6 +51,28 @@ class AppInstallBroadcastReceiver : BroadcastReceiver() {
                     notificationManager.notify(sessionId, notification)
                 }
             }
+            PackageInstaller.STATUS_SUCCESS -> {
+                val notification =
+                    NotificationCompat.Builder(context, Accrescent.UPDATED_FINISHED_CHANNEL)
+                        .setSmallIcon(R.drawable.ic_baseline_file_download_done_24)
+                        .setContentTitle(context.getString(R.string.update_finished))
+                        .setContentText(context.getString(R.string.update_success, sessionInfo?.appPackageName))
```

Comment:
From my testing, `sessionInfo` seems to sometimes (always?) resolve to `null`, resulting in the message "null has been updated successfully." To reproduce:

1. Install Connect You < 2.4 from [GitHub releases](https://github.com/Bnyro/ConnectYou/releases).
2. Update Connect You from within Accrescent.
3. A notification with the erroneous message appears.

Reply:
Hmm, unfortunately I don't have any experience with the `PackageManager` so I've no clue why that happens. I decided to now pass the app's name directly to the broadcast receiver, which now works properly. However, it still seems to always use the app's package Name and not it's label although we're explicitely grabbing that from the `PackageManager` :/

Reply:
> However, it still seems to always use the app's package Name and not it's label although we're explicitely grabbing that from the `PackageManager`

Can you clarify? I don't understand.

I think it would be cleaner to pull the package label from the [database](https://github.com/accrescent/accrescent/blob/0534277584ad49a6dd9b5d9e34f3026570116535/app/src/main/java/app/accrescent/client/data/db/App.kt#L10) by injecting `RepoDataRepository` into `AppInstallStatusReceiver` rather than extracting it from the package info and passing it via extras.

Reply:
> Can you clarify? I don't understand. It appears to work fine now from my testing.
>
It always shows the package name (e.g. com.bnyro.contacts) instead of the actual app name (Connect You). 
> However, I think it would be cleaner to pull the package label from the [database](https://github.com/accrescent/accrescent/blob/0534277584ad49a6dd9b5d9e34f3026570116535/app/src/main/java/app/accrescent/client/data/db/App.kt#L10) by injecting `RepoDataRepository` into `AppInstallStatusReceiver` rather than extracting it from the package info and passing it via extras.
>
Good idea, that will also resolve my issue!